stuffer/s2n_stuffer_file.c

```diff
@@ -44,8 +42,9 @@ int s2n_stuffer_recv_from_fd(struct s2n_stuffer *stuffer, const int rfd, const u
 
     /* Record just how many bytes we have written */
     POSIX_ENSURE(r <= UINT32_MAX, S2N_ERR_INTEGER_OVERFLOW);
-    POSIX_GUARD(s2n_stuffer_skip_write(stuffer, (uint32_t)r));
-    if (bytes_written != NULL) *bytes_written = r;
+    POSIX_GUARD(s2n_stuffer_skip_write(stuffer, (uint32_t) r));
+    if (bytes_written != NULL)
+        *bytes_written = r;
```

Comment:
Is there any way to enforce to always have curly braces on `if`, `for` and `while`? (see https://www.imperialviolet.org/2014/02/22/applebug.html)

Reply:
Looks like `InsertBraces: true` will do that (only available in clang 15)

Reply:
Yes. But: see the warning "InsertBraces" here: https://clang.llvm.org/docs/ClangFormatStyleOptions.html

Reply:
I'll manually insert them.